vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericWaitSteps.java

```diff
@@ -132,6 +142,57 @@ public String toString()
         });
     }
 
+    /**
+     * Waits for an <b><i>element</i></b> with the specified <b>locator</b> to stop moving
+     * @param locator locator to locate element
+     */
+    @When("I wait until element located by `$locator` stops moving")
+    public void waitForElementStopsMoving(Locator locator)
+    {
+        uiContext.getOptionalSearchContext().ifPresent(context -> {
+            locator.getSearchParameters().setWaitForElement(false);
+            IExpectedSearchContextCondition<Boolean> condition = new IExpectedSearchContextCondition<>() {
+                private Point lastPosition;
+                private boolean elementPresent = true;
+                private WebElement webElement = findWebElementAndAssertElementExists(context, locator);
+
+                @Override
+                public Boolean apply(SearchContext context)
+                {
+                    try
+                    {
+                        Point currentPosition = webElement.getLocation();
+                        boolean samePosition = currentPosition.equals(lastPosition);
+                        if (!samePosition)
+                        {
+                            LOGGER.info("The element is moved at '{}'", currentPosition);
+                        }
+                        lastPosition = currentPosition;
+                        elementPresent = true;
+                        return samePosition;
+                    }
+                    catch (StaleElementReferenceException e)
+                    {
+                        if (!elementPresent)
+                        {
+                            throw new StaleElementReferenceException(e.getMessage(), e);
+                        }
+                        elementPresent = false;
+                        webElement = findWebElementAndAssertElementExists(context, locator);
```

Comment:
I believe we shouldn't validate element exists at retry: if element is found, then proceed, otherwise throw the caught exception